### PR TITLE
Added Lw3

### PR DESCRIPTION
**Pull request by RylkovKirill**
https://anki-9bd54.web.app/

Hello, I changed the design of the "start-collection" page. Initially there were two pages with the sides of the card. Now this is one page with the animation of the card flip. 
user can already register)

Some pages cannot yet be navigated using elements:
https://anki-9bd54.web.app/#/start-collection/1
https://anki-9bd54.web.app/#/edit-collection/1
https://anki-9bd54.web.app/#/edit-card/1

### CODE REVIEW

File: Anki/public/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <button onclick="logout()" id="logoutButton" class="button-animated button-secondary">Log out</button>
+        </div>
+    </header>
+    <main id="root">
+    </main>
+    <footer>
+        <div class="container">2021-Anki-<a href="/#/privacy">Privacy</a></div>
+        <div class="container">
+            <label class="theme-switch" for="themeCheckbox">
+                <input type="checkbox" id="themeCheckbox"/>
```
Comment (gmltA) [inline Anki/public/index.html]: When the checkbox is completely hidden via `display: none` it is not accessible via keyboard by users. Try something else instead.

File: Anki/public/assets/js/auth.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+var firebaseConfig = {
```
Comment (gmltA) [inline Anki/public/assets/js/auth.js]: You'll have to put all JS into new PR as well.

**Review by gmltA — COMMENTED** [review]
That's a <kbd>6</kbd> for now.

Fix my remarks, define more styles for interactive elements (see [results grading](https://github.com/gmltA/FrontEnd_tasks#result-grading-1)) to improve it.

File: Anki/public/index.html (hunk 1 above)
Comment (RylkovKirill) [reply Anki/public/index.html]: I changed `display: none` to `opacity: 0` and when the check-box has focus the slider has an animated  box-shadow.
The same problem was with the checkbox for the menu.

**Comment by RylkovKirill** [comment]
I also added styles for the interactive elements:
1) `a` - color change
2) `a` in footer - change color and scale icon
3) `button` - scale and add shadow.
4) `select` - add a shadow.

https://anki-9bd54.web.app/

File: Anki/public/assets/js/auth.js (hunk 2 above)
Comment (RylkovKirill) [reply Anki/public/assets/js/auth.js]: Removed all js code. At least i hope i got it right)

**Review by gmltA — COMMENTED** [review]
Good. It is <kbd>8</kbd> for now. Tell me if you want no improve it.

**Comment by RylkovKirill** [comment]
I agree with 8.

**Review by gmltA — APPROVED** [review]

**Pull request merged**